remixer_storage/src/main/java/com/google/android/libraries/remixer/storage/FirebaseRemoteControllerSyncer.java

```diff
@@ -264,18 +264,25 @@ public void addSharingStatusListener(@NonNull SharingStatusListener listener) {
   public Intent getShareLinkIntent() {
     Intent intent = new Intent();
     intent.setAction(Intent.ACTION_SEND);
-    String url = String.format(
-        Locale.getDefault(),
-        "https://%s.firebaseapp.com/%s",
-        getUrlPrefix(),
-        remoteId);
+    String url = getRemoteUrl();
     String message =
         applicationContext.getResources().getString(R.string.shareLinkFormat, url);
     intent.putExtra(Intent.EXTRA_TEXT, message);
     intent.setType("text/plain");
     return intent;
   }
 
+  /**
```

Comment:
Actually yes, it was a tiny thing that was missing and I noticed in documentation, I meant to offer this from the beginning. Writing another CL seemed like overdoing it.